OnionBrowser/Browsing/Tab.swift

```diff
@@ -197,7 +209,6 @@ class Tab: UIView {
 		if url == URL.start {
 			Bookmark.updateStartPage()
 		}
-
```

Comment:
The space between the text is import for readability. Please don't mess with my line breaks.
These are part of the reason why you could read the code and make the changes easily.

I built a career on fixing other peoples mess, and this is part of why I succeed where others give up.